dashboard/src/components/Header/UISelector/UISelector.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+const HEx = React.lazy(() => import("./HEx"));
```

Comment:
I see it matches the filename, but was that a typo? (the 'E' rather than 'e')

Reply:
nope, the design system is called "HEx": https://design.bitnami.com/